doc_src/gear_developers/limitations.md

```diff
@@ -0,0 +1,13 @@
+# Limitations
+
+There are certain limitations in Antikythera.
+We list some major limitations here. But this does not cover all the limitations of Antikythera. Please also read other parts of the documentation when you implement gear.
+
+- Execution time of web/g2g request
+    - Set via `GEAR_ACTION_TIMEOUT`. Please refer [here](https://hexdocs.pm/antikythera/development_environment.html#environment-variables-to-tweak-behavior-of-antikythera).
+- Execution time of AsyncJob
```

Comment:
According to [this page](https://github.com/access-company/antikythera/blob/master/doc_src/gear_developers/async_job.md), in the English context, it is "async job" and the module name is "AsyncJob".
If you want to emphasize aspects of the module, I think you should use single quotes ( `AsyncJob`).
```suggestion
- Execution time of async job
```

Reply:
I should have used "async job".
I fixed in c919a54f719be8f28707e2a9b00fb8c9af816cd7 .